src/inc/StringStream.hpp

```diff
@@ -0,0 +1,68 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+#pragma once
+
+#include <Exceptions.hpp>
+#include <StreamBase.hpp>
+
+#include <sstream>
+
+namespace MSIX {
+
+    class StringStream final : public StreamBase

```

Comment:
I would suggest a secondary interface or direct usage of this class to add a helper that simply takes a string for writing.  Or even a static/standalone helper to simplify the call sites.

Reply:
Actually, a standalone helper is best, so it could target any IStream.  Oh to have extension functions syntactic sugar.

---
In reply to: [278223918](https://github.com/Microsoft/msix-packaging/pull/95#discussion_r278223918) [](ancestors = 278223918)

Reply:
Helper in StreamHelper.hpp

---
In reply to: [278233164](https://github.com/Microsoft/msix-packaging/pull/95#discussion_r278233164) [](ancestors = 278233164,278223918)